docs/reference/integrated-tools/kobotoolbox/best-practices.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 2
+---
+
+# Best Practices and Usage Tips
+
+This page covers practical lessons learned from using KoboToolbox in real-world scenarios, including common pitfalls and their solutions.
+
+## Browser Compatibility and Data Collection Methods
+
+### Avoid Enketo Web Forms on iOS Safari for Media Submissions
+
+**Issue**: Using Enketo web forms in Safari on iPhone as a workaround for KoboToolbox's lack of a native iOS app can cause significant problems with file uploads.
+
+**Best Practice**: 
+- Use Android devices with the official KoboCollect app for surveys requiring media attachments (photos, audio, video)
+- If iOS devices are necessary, consider alternative data collection workflows that don't rely on media submissions through the browser
```

Comment:
I am not sure that the issue is connected to media submissions, but I feel it is connected with LocalStorage... haven't dag into it enough though